src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt

```diff
@@ -27,6 +37,7 @@ class RESTZaakHistoriePartialUpdateConverter @Inject constructor(
             datumTijd = auditTrail.aanmaakdatum
             door = auditTrail.gebruikersWeergave
             toelichting = auditTrail.toelichting
+            this.actie = actie
```

Comment:
Do we test this ?

Reply:
Meant to add this as a draft PR as well. I will at the very least extend the existing integration tests.

Reply:
We do now :-)